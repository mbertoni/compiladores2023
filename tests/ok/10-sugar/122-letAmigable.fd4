let (f100:Nat) = fun (x:Nat) -> (x + 2) 
let print100:Nat = print "f 98 = " (f100 98)