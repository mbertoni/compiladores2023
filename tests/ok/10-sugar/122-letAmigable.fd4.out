f 98 = 100
